### PR TITLE
updated experiment doc for networking changes

### PR DESCRIPTION
**Pull request by kalantar**

### CODE REVIEW

File: content/reference/experiment.md [hunk 1]
```diff
@@ -83,9 +74,34 @@ service:
...
+*gateway* | string | The name of gateway to which this host is attached. | yes
+
+An example of the `networking` subsection of an experiment object is as follows:
+
+```yaml
+  networking:
+    hosts:
+      - name: bookinfo.example.com
+        gateway: bookinfo-gateway
+  id: bookinfo
```
Comment (kimikowang) [inline content/reference/experiment.md:104]: `id` is out of its parent section.

**Comment by kimikowang** [comment]
Is it possible to remove the type column from each table? I think that might be unnecessary.

File: content/reference/experiment.md (hunk 1 above)
Comment (kalantar) [reply content/reference/experiment.md]: Good catch. Fixed.

**Comment by kalantar** [comment]
> Is it possible to remove the type column from each table? I think that might be unnecessary.

I suppose so. I find it helpful for some of the later items especially when there is an enum.

**Comment by sriumcp** [comment]
On types, here's how Istio documents CRDs: https://istio.io/latest/docs/reference/config/networking/virtual-service/. Experiment crd description is directly inspired by Istio (table format here = Istio's table format).

**Comment by kimikowang** [comment]
> On types, here's how Istio documents CRDs: https://istio.io/latest/docs/reference/config/networking/virtual-service/. Experiment crd description is directly inspired by Istio (table format here = Istio's table format).

Those types are hyperlinks. But we do not have links for the types in the doc.

**Review by kimikowang — APPROVED** [review]

**Comment by sriumcp** [comment]
Some of those types are hyperlinks in Istio docs. Not all (for e.g., there are strings and string arrays in types without types).

Yes, a PR with hyperlinks in types (where they exist) would be a nice addition.

**Pull request merged**